charts/ambassador/templates/service-admin.yaml

```diff
@@ -3,6 +3,9 @@ kind: Service
 metadata:
   labels:
     service: {{ .Values.global.ambassador_service_name }}-admin
+    # This labels the service as the type 'ambassador' even if the name is something else. It's used
```

Comment:
Services typically use the app=(app-name) strategy to select and switch on. When I as a user am requesting the service app=ambassador, how do I know which ambassador is which?